plugins/kubernetes/app/models/kubernetes/role_validator.rb

```diff
@@ -103,6 +104,12 @@ def self.keep_name?(e)
 
     private
 
+    def validate_load_balancer
+      return unless allowed = ENV["KUBERNETES_ALLOWED_LOAD_BALANCER_NAMESPACE"]
+      return if @elements.none? { |e| e[:kind] == "LoadBalancer" && e.dig(:metadata, :namespace) != allowed }
```

Comment:
Would prefer to have it as a list of allowed namespaces rather than just a single namespace in case, we want to stick to this for a while and use across multiple namespaces.